.gitignore

```diff
@@ -1,3 +1,15 @@
+# Migrations
+migrations/*
+imager_images/migrations/*
+imager_profile/migrations/*
```

Comment:
You actually want to keep your migration files in the repository, especially once they are finalized.  They are part of the app.